google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineRuntimeInstance.java

```diff
@@ -70,8 +72,11 @@ public void deploy(@NotNull final DeploymentTask<AppEngineDeploymentConfiguratio
 
     final AppEngineDeployAction deployAction;
     AppEngineDeploymentConfiguration deploymentConfig = task.getConfiguration();
-    File deploymentSource = deploymentConfig.isUserSpecifiedArtifact() ?
-        new File(deploymentConfig.getUserSpecifiedArtifactPath()) : task.getSource().getFile();
```

Comment:
Yes we did, and I think this was one place where the refactoring was overlooked. I'll double check and update it.